### PR TITLE
Add Chef Infra Client 15.10 release notes

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: content/release_notes.md [hunk 1]
```diff
@@ -1899,7 +1930,7 @@ Chef Infra Client is now tested against the following platforms with packages av
 ### Retired Platforms
 
   - Chef Infra Clients packages are no longer produced for Windows 2008 R2 as this release reached its end of life on Jan 14th, 2020.
-  - Chef Infra Client packages are no longer produced for RHEL 6 on the s390x platform.
+  - Chef Infra Client packages are no longer produced for RHEL 6 o n the s390x platform.
```
Comment (phiggins) [inline content/release_notes.md]: ?

Comment (phiggins) [inline content/release_notes.md]:
```suggestion
  - Chef Infra Client packages are no longer produced for RHEL 6 on the s390x platform.
```

**Review by phiggins — APPROVED** [review]

File: content/release_notes.md (hunk 1 above)
Comment (tas50) [reply content/release_notes.md]: No clue what happened there. I'm going to blame my kid finding my keyboard

Comment (phiggins) [reply content/release_notes.md]: "kid typing" is the new "fat finger" :)

**Review by IanMadd — APPROVED** [review]

**Pull request merged**